src/App.jsx

```diff
@@ -1,65 +1,63 @@
-import React, { Component } from 'react';
-import './App.css';
-import { Button } from './components/Button';
-import { Input} from './components/Input';
-import { ClearButton} from './components/ClearButton';
+import React, { useState } from 'react';
+import styles from './App.module.css';
+import Button from './components/Button';
+import Input from './components/Input';
+import ClearButton from './components/ClearButton';
 import * as math from 'mathjs';
 
-class App extends Component {
-  constructor(props) {
-    super(props);
+const App = props => {
+  const [inputState, setInputState] = useState({
```

Comment:
you don't need to use object, it's good to use simple variable types

`const [input, setInput] = useState("");`